test/test-functions

```diff
@@ -99,11 +103,18 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    PARAMS+="systemd.mask=systemd-networkd.service "
```

Comment:
SUSE is currently using wicked for network setup and systemd-networkd is disabled. This is just an attempt to avoid interference between the 2 methods in the tests. But if you prefer, I can try to disable systemd-networkd instead of masking it. Again, this is done to be as close as possible to real life scenarios in the tests.